src/main/scala/rifs/business/tables/OpportunityTables.scala

```diff
@@ -110,6 +110,15 @@ class OpportunityTables @Inject()(val dbConfigProvider: DatabaseConfigProvider,
   lazy val sectionC = Compiled(sectionQ _)
 
   override def saveSectionDescription(id: OpportunityId, sectionNo: Int, description: Option[String]): Future[Int] = db.run {
-    sectionQ(id, sectionNo).map(r=> r.text).update(description)
+    sectionQ(id, sectionNo).map(r => r.text).update(description)
+  }
+
+  override def reset(): Future[Unit] = {
```

Comment:
I'd hope that the tests don't rely on specific ids. In fact, this is probably more for user research, who also use the reset between sessions. Might be less surprising for test users if the newly created opportunity is number 2 when duplicating from 1, rather than some "random" number.